From bb395170c64d3e7d0387811f92b6ff25fc66d069 Mon Sep 17 00:00:00 2001
From: b07117 <b07117@freescale.com>
Date: Wed, 21 Dec 2011 21:32:02 +0800
Subject: ENGR00170747 gpu driver : add AXI BUS ERROR message

AXI BUS ERROR may occur in very low possibility,
this debug message exist before 4.4.2, but removed in 4.6.x,
need add it back to trace critical gpu issue

Signed-off-by: Li Xianzhong <b07117@freescale.com>
Signed-off-by: Loren Huang <b02279@freescale.com>
Acked-by: Lily Zhang
---
 .../arch/XAQ2/hal/kernel/gc_hal_kernel_hardware.c  |    1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/mxc/gpu-viv/arch/XAQ2/hal/kernel/gc_hal_kernel_hardware.c b/drivers/mxc/gpu-viv/arch/XAQ2/hal/kernel/gc_hal_kernel_hardware.c
index 4b89efe..7eb8fbd 100644
--- a/drivers/mxc/gpu-viv/arch/XAQ2/hal/kernel/gc_hal_kernel_hardware.c
+++ b/drivers/mxc/gpu-viv/arch/XAQ2/hal/kernel/gc_hal_kernel_hardware.c
@@ -2296,6 +2296,7 @@ gckHARDWARE_Interrupt(
 
         if (data & 0x80000000)
         {
+		gckOS_Print("!!!!!!!!!!!!! AXI BUS ERROR !!!!!!!!!!!!!\n");
             gcmkTRACE_ZONE(gcvLEVEL_ERROR, gcvZONE_HARDWARE, "AXI BUS ERROR");
         }
 
-- 
1.7.9.5

